### ISSUE TITLE
How to download LSDIR dataset?

### ISSUE THREAD

**Issue by lovecq-23** [kunncheng/DiT-SR#5]
The link for downloading the LSDIR dataset has expired. Could you please update the download link? And I am very confused about how to place the datasets used in the training and testing phases in the project after downloading them? By the way, thank you very much for your great work!

![Image](https://github.com/user-attachments/assets/bf5d672a-0782-4c0f-9534-1791c0359d89)

**Comment by kunncheng** [comment]
You can download the LSDIR dataset from https://huggingface.co/ofsoundof/LSDIR.

For the training set, you can put anywhere, but a [txt file](https://github.com/kunncheng/DiT-SR/blob/main/data_list/LSDIR_train.txt) containing the file paths needs to be generated.
For the test set, you can just edit the input folder of [inference script](https://github.com/kunncheng/DiT-SR/blob/7a83963d37c8011a8944803dfe388d360a87e613/test_realsr.sh#L6).

**Comment by lovecq-23** [comment]
> You can download the LSDIR dataset from https://huggingface.co/ofsoundof/LSDIR.
> 
> For the training set, you can put anywhere, but a [txt file](https://github.com/kunncheng/DiT-SR/blob/main/data_list/LSDIR_train.txt) containing the file paths needs to be generated. For the test set, you can just edit the input folder of [inference script](https://github.com/kunncheng/DiT-SR/blob/7a83963d37c8011a8944803dfe388d360a87e613/test_realsr.sh#L6).

I found a file named LSDIR_train.txt in the data_list folder. Do you mean I need to generate a similar txt file to replace it, or modify its contents directly? The content of this file is as follows, does it contain the paths of all images?

![Image](https://github.com/user-attachments/assets/ef239905-3a8d-41ee-9848-fff089d7cb54)

**Comment by kunncheng** [comment]
This txt file contains all the training images of LSDIR. You can decide to modify or generate from scratch by yourself. The key is to ensure that the paths within the file are consistent with the paths of your dataset.

**Comment by lovecq-23** [comment]
> This txt file contains all the training images of LSDIR. You can decide to modify or generate from scratch by yourself. The key is to ensure that the paths within the file are consistent with the paths of your dataset.

Thank you very much for your help! I will try it.
